src/styles/objects.CourseModal.scss

```diff
@@ -0,0 +1,25 @@
+@import "settings.constants.scss";
+
+.preview-modal-body {
+  display: flex;
+}
+
+.course-preview-wrap {
+  width: 112px;
```

Comment:
lots of magic numbers - should any of them be constants?

Reply:
A lot of it is just me eyeing it up, which is kinda terrible in it's own right, but I'm no designer. We can pull some of them out into constants in their own right, but I don't know how much value that would have, unless more than one thing would be referencing it.